### ISSUE TITLE
Feature Request: One way sync?

### ISSUE THREAD

**Issue by KamChanLiu** [poetic/meteor-rest-sync#1]
It would be great to be able to specify a one way sync (i.e. a mapIn but no mapOut).
Is there a way to add a collection with this option under the current implementation?

**Comment by jimmiebtlr** [comment]
I don't believe that is possible currently.  Should probably be a flag that can be checked in the start function dependent on collection.
